crux-bench/src/crux/bench/tpch_test.clj

```diff
@@ -23,19 +23,20 @@
                         (boolean actual)))))))
 
 (defn run-tpch-test [node {:keys [scale-factor] :as opts}]
-  (bench/with-bench-ns :tpch-test
-    (bench/with-crux-dimensions
-      (bench/run-bench :ingest
-        (bench/with-additional-index-metrics node
-          (tpch/load-docs! node scale-factor tpch/tpch-entity->pkey-doc)))
-
-      ;; TODO we may want to split this up, à la WatDiv, so that we can see if
-      ;; specific queries are slower than our comparison databases
-      (bench/run-bench :queries
-        {:success? (run-tpch-queries node opts)})
-
-      (bench/run-bench :queries-warm
-        {:success? (run-tpch-queries node opts)}))))
+  (let [scale-factor (or scale-factor 0.01)]
```

Comment:
Think this might be a bit redundant as we are providing it with a default of 0.01 within `bench.clj`?